### PR TITLE
New commit

### PR DESCRIPTION
**Pull request by Helgi4**
It's my homework for lessons 5 & 6.

### CODE REVIEW

File: src/MyClassSixth2.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import java.util.HashMap;
+
+public class MyClassSixth2 {
+
+    public static void main(String[] args) {
+        ArrayList<String> books = new ArrayList<>();
+        books.add("Bible"); //+
+        books.add("Will");
+        books.add("The Great Gatsby");
+        books.add("Nineteen Eighty-four"); //1984 - real cool book
```
Comment (DarynaHorobei97) [inline src/MyClassSixth2.java:11]: I read, agree)

File: src/Animal/Animal.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
+package Animal;
+
+class Animal {
+    int eyes;
```
Comment (DarynaHorobei97) [inline src/Animal/Animal.java:4]: why not private?

**Review by DarynaHorobei97 — COMMENTED** [review]
The 5th HW - where is the 1st task - I found the 2nd task only
The 6th HW - good job!

**Review by DarynaHorobei97 — COMMENTED** [review]
Also do not merge PR before review! Firstly, I need to review, after that you need to make some changes. Only after that you should merge your code to main!

File: src/Animal/Animal.java [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    }
+
+    public void jump() {
+        System.out.println("Animals.Pet runs and jumps");
+    }
+}
+
+class Dog extends Pet {
+    public Dog(String name) {
+        System.out.println("I'm dog and my name is: " + name);
```
Comment (DarynaHorobei97) [inline src/Animal/Animal.java:40]: It's a good idea to call parent's constructor - super();

**Comment by Helgi4** [comment]
> Also do not merge PR before review! Firstly, I need to review, after that you need to make some changes. Only after that you should merge your code to main!

Got it, sorry

**Comment by Helgi4** [comment]
> The 5th HW - where is the 1st task - I found the 2nd task only The 6th HW - good job!

Sh*t, I miss the first task. (( 
My apologies!

**Pull request merged**